apps/api/src/app/inbox/utils/notification-mapper.ts

```diff
@@ -0,0 +1,69 @@
+import type { MessageEntity } from '@novu/dal';
+import { ButtonTypeEnum } from '@novu/shared';
+
+import type { InboxNotification, Subscriber } from './types';
+
+const mapSingleItem = ({
+  _id,
+  content,
+  read,
+  archived,
+  createdAt,
+  lastReadDate,
+  archivedAt,
+  channel,
+  subscriber,
+  actorSubscriber,
+  actor,
+  cta,
+  tags,
+}: MessageEntity): InboxNotification => {
+  const to: Subscriber = {
+    id: subscriber?._id ?? '',
+    firstName: subscriber?.firstName,
+    lastName: subscriber?.lastName,
+    avatar: subscriber?.avatar,
+    subscriberId: subscriber?.subscriberId ?? '',
+  };
+  const primaryCta = cta.action?.buttons?.find((button) => button.type === ButtonTypeEnum.PRIMARY);
+  const secondaryCta = cta.action?.buttons?.find((button) => button.type === ButtonTypeEnum.SECONDARY);
+
+  return {
+    id: _id,
+    // subject: subject,
```

Comment:
Let's add a todo to support this next week as it directly affects the UI of the new Inbox.